jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -337,6 +337,68 @@ def check_eigenvalue_is_in_array(eigenvalue, eigenvalues_array):
     self.ConvertAndCompare(harness.dyn_fun, operand,
                            custom_assert=custom_assert)
 
+  @primitive_harness.parameterized(primitive_harness.lax_linalg_eigh)
+  def test_eigh(self, harness: primitive_harness.Harness):
+    operand = harness.dyn_args_maker(self.rng())[0]
+    lower = harness.params["lower"]
+    # Make operand self-adjoint
+    operand = (operand + np.conj(np.swapaxes(operand, -1, -2))) / 2
+    # Make operand lower/upper triangular
+    triangular_operand = np.tril(operand) if lower else np.triu(operand)
+    dtype = harness.params["dtype"]
+
+    if dtype in [np.float16]:
```

Comment:
I think that we should start to put these in the primitive_harnesses, e.g., filter these cases out.